### PR TITLE
Changed Accordion, Datepicker, Dropdown, FileUpload component from Vue 2 to Vue 3

### PR DESCRIPTION
**Pull request by Palaksharma23**
I have changed Accordion, Datepicker, Dropdown, FileUpload component from Vue 2 to Vue 3

@HarshCasper may you please once review.

### CODE REVIEW

**Review by HarshCasper — APPROVED** [review]

**Review by YashKandalkar — COMMENTED** [review]
@Palaksharma23 Thanks for working on this!
I think we can remove the Accordion component, and then merge this in.

File: flint.ui/src/components/Accordion/Accordion.vue [hunk 1]
```diff
@@ -5,18 +5,22 @@
 </template>
 
 <script>
```
[the remark this answers is not in the document]
Comment (Palaksharma23) [reply flint.ui/src/components/Accordion/Accordion.vue:7]: Yes, we are not using it anywhere in the project. I just changed it to not delete any file by myself.  
I will remove the Accordion folder :)

**Review by YashKandalkar — APPROVED** [review]
LTGM! @Palaksharma23 Thanks! :rocket:

**Comment by YashKandalkar** [comment]
@Palaksharma23 You can fix the lining errors with `yarn lint`

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**